tgui/packages/tgui/interfaces/Terminal.js

```diff
@@ -0,0 +1,80 @@
+import { useBackend } from '../backend';
+import { Window } from '../layouts';
+import { Box, Input, Button, Section } from 'tgui/components';
+import { ColorBox, Flex } from '../components';
+
+export const Terminal = (props, context) => {
+  const { act, data } = useBackend(context);
+  const peripherals = data.peripherals || [];
+  const { textInput } = props;
+
+  const {
+    fdisk,
+    idcard,
+    displayHTML,
+    TermActive,
+    windowName,
+    user,
+    fontColor,
+    bgColor,
+    diskdrive,
+  } = data;
+
+  return (
+    <Window
+      theme="retro-dark"
+      title={windowName}
+      width="380"
+      fontFamily="Consolas"
+      font-size="10pt"
```

Comment:
Any specific reason for using a smaller font? The window is pretty small, would it make more sense to leave it as default and increase the window size a bit to compensate?